### PR TITLE
Code Smells - Remove unused variables

### PR DESCRIPTION
**Pull request by tiagograf-AOT**
# Fixes or Implements [AB#10444](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/10444)

-   [x] Enhancement
-   [ ] Bug
-   [ ] Documentation

## Description

Remove unused variables
Remove commented out code

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

-   [x] Unit Tests Updated
-   [ ] Functional Tests Updated
-   [ ] Not Required

### Steps to Reproduce

If this is a bug, please provide details on how to reproduce the code.

### UI Changes

YES | NO

### Browsers Tested

-   [ ] Chrome
-   [ ] Safari
-   [ ] Edge
-   [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant. Include any specialized deployment steps here.

### CODE REVIEW

**Review by MikeLyttle — APPROVED** [review]

File: Apps/AdminWebClient/src/ClientApp/src/services/httpDelegate.ts [hunk 1]
```diff
@@ -128,17 +128,6 @@ export default class HttpDelegate implements IHttpDelegate {
                     console.log(errorMessage);
                     return reject(errorMessage);
                 });
-            // TODO: Axios has bug with the delete method not using data fields.
```
Comment (FuriousLlama) [inline Apps/AdminWebClient/src/ClientApp/src/services/httpDelegate.ts:131]: Lets keep TODOS until they are solved or dismissed

**Review by FuriousLlama — CHANGES_REQUESTED** [review]
Todos need to be addressed

File: Apps/AdminWebClient/src/ClientApp/src/services/httpDelegate.ts (hunk 1 above)
Comment (tiagograf-AOT) [reply Apps/AdminWebClient/src/ClientApp/src/services/httpDelegate.ts:131]: created a bug https://qslvic.visualstudio.com/Health%20Gateway/_workitems/edit/10506
Todos must be used for work in progress only and should not pass the code review/merged to dev

Comment (FuriousLlama) [reply Apps/AdminWebClient/src/ClientApp/src/services/httpDelegate.ts:131]: > created a bug https://qslvic.visualstudio.com/Health%20Gateway/_workitems/edit/10506
> Todos must be used for work in progress only and should not pass the code review/merged to dev

That sounds good as well. We can block PR if they have TODOs on them

**Comment by FuriousLlama** [comment]
> Todos need to be addressed

Ticket logged 
https://qslvic.visualstudio.com/Health%20Gateway/_workitems/edit/10506

**Review by FuriousLlama — APPROVED** [review]

**Pull request merged**